client/template.js

```diff
@@ -22,6 +22,9 @@ const template = async function(name, title='', props = {}){
 		<body>
 			<main id="reactRoot">${require(`../build/${name}/ssr.js`)(props)}</main>
 			<script src=${`/${name}/bundle.js`}></script>
+			<script>
+				sessionStorage.setItem('version', '${props.version}');
+			</script>
```

Comment:
We already have access to the version via `global.version` set in Homebrew.jsx (the root React container). This is how we access it to display the version number on the nav bar.

We also set a handful of other globals there. That may or may not be the best place to put them, but at least for Version let's have it in only one location to keep our data? If you want to have it in Session Storage, let's change the nav bar to also use session storage. Otherwise jus use `global.version` and remove this from the template.

(also, I think I prefer to not handle this inside the template; I've kind of intended it to just load and boot the application rather than setting variables and stuff.)

Reply:
> We already have access to the version via `global.version` set in Homebrew.jsx (the root React container). This is how we access it to display the version number on the nav bar.

Ah I didn't realize that, yeah I'll get rid of the session storage stuff